stdlib/betaTransformations.hk

```diff
@@ -0,0 +1,14 @@
+def invBeta(a prob, b prob):
+	X <~ beta(a,b)
+	return X / (1 - X)
+
+def stdUniform():
```

Comment:
The fact that `stdUniform` is equivalent to a `beta` should not be part of the code of the standard library, but instead should be in a test.